app/Config/Entity.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Config;
+
+use CodeIgniter\Config\BaseConfig;
+use CodeIgniter\Entity\Cast\ArrayCast;
+use CodeIgniter\Entity\Cast\BooleanCast;
+use CodeIgniter\Entity\Cast\CSVCast;
+use CodeIgniter\Entity\Cast\DatetimeCast;
+use CodeIgniter\Entity\Cast\FloatCast;
+use CodeIgniter\Entity\Cast\IntegerCast;
+use CodeIgniter\Entity\Cast\JsonCast;
+use CodeIgniter\Entity\Cast\ObjectCast;
+use CodeIgniter\Entity\Cast\StringCast;
+use CodeIgniter\Entity\Cast\TimestampCast;
+use CodeIgniter\Entity\Cast\URICast;
+
+class Entity extends BaseConfig
+{
+    /**
+     * Convert handlers
+     *
+     * @var array
+     */
+    public $castHandlers = [
```

Comment:
New properties should be typed since the next version will be minimum 7.4:
```suggestion
     */
    public array $castHandlers = [
```